clients/spark/build.sbt

```diff
@@ -55,7 +55,7 @@ libraryDependencies ++= Seq(
   "com.azure" % "azure-storage-blob" % "12.9.0",
   "com.azure" % "azure-storage-blob-batch" % "12.7.0",
   "com.azure" % "azure-identity" % "1.2.0",
-  "com.amazonaws" % "aws-java-sdk-bundle" % "1.12.194" % "provided",
+  "com.amazonaws" % "aws-java-sdk-bundle" % "1.12.367" % "provided",
```

Comment:
According to the [emr-7.0.0](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-700-release.html) AWS SDK version is `1.12.569`, why did you set lower here?